ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -882,6 +892,19 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         }
     }
 
+    /// The custom view on the leading edge of the `customView` UIView. Does not display if layoutType is `.oneLine`.
```

Comment:
why not in .oneline?

Reply:
Specs only want it in multi-line

Reply:
Actually I suppose if I allow flexibility for the UIView, I can allow flexibility in any layoutType?

Reply:
yeah it feels little strange. "you have all the knobs! jk... only certain times"